Sources/Shared/ViewModel.swift

```diff
@@ -36,6 +37,8 @@ public struct ViewModel: Mappable {
 
   /// The index of the ViewModel when appearing in a list, should be computed and continuously updated by the data source
   public var index = 0
+  /// An optional identifier for your data
+  public var identifier: Int?
```

Comment:
`Identifier` works for me too, I just remember that usually we had `meta["id"]` in our code.